src/azure-cli/azure/cli/command_modules/monitor/operations/autoscale_settings.py

```diff
@@ -45,13 +46,27 @@ def autoscale_create(client, resource, count, autoscale_name=None, resource_grou
                 notification.email.custom_emails.append(email)
         elif isinstance(action, WebhookNotification):
             notification.webhooks.append(action)
+    predictive_policy = None
+    if scale_mode is not None and scale_look_ahead_time is not None:
+        predictive_policy = PredictiveAutoscalePolicy(
+            scale_mode=scale_mode,
+            scale_look_ahead_time=scale_look_ahead_time
+        )
+    elif scale_mode is not None:
+        predictive_policy = PredictiveAutoscalePolicy(
+            scale_mode=scale_mode
+        )
+    else:
+        logger.warning('PredictiveAutoscalePolicy scale_mode is required')
```

Comment:
Is predictive_policy required or not? If it's required, you should raise error here, else the warning log is useless.

Reply:
not required， removed warning